.github/workflows/validation-opened-prs.yml

```
@@ -0,0 +1,83 @@
+      - reopened
+
+permissions: write-all
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      validation-cache-key: ${{ steps.validation-cache.outputs.key }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)